### DISCUSSION TITLE
Strange behavior Tap-Hold-Move gesture

### DISCUSSION THREAD

**Discussion by deevroman** [streetcomplete/StreetComplete#4414] [Feedback]
Maybe I didn't notice it, but it seems to me that when I let go of my finger, the card didn't come closer. Now you can get rid of this only by releasing your finger a little slower.

Has this happened before?

https://user-images.githubusercontent.com/25708359/192268924-eb37f193-a747-4eb6-9a1b-5281e07747f7.mp4

**Comment by HolgerJeromin** [comment]
That is in fact a zoom-with-only-one-finger feature and implemented in surprising many map frameworks (like OSMand, Google Maps) :-)

touchdown, touchup, (fast) touchdown, touchmove is a zoom

**Reply by deevroman** [reply]
I know about this:) Pay attention at the beginning of the video when you release your finger, the map is getting closer. Although I was only distancing it.
